public/static/gameScene.js

```diff
@@ -447,6 +471,38 @@ class GameScene extends Phaser.Scene {
 
     }
 
+    displayMissileCount(self, id, positionX, newAmount, maxAmount, regenTime) {
+        const positionY = 575;
+        const maxDisplayTimeMs = 5000;
+        let myGroup = this.add.group();
+        self.missileCounterUIs[id] = myGroup;
+        const missileSprite = self.add.sprite(positionX - 45, positionY, 'missile').setDisplaySize(20,30).setDepth(100);
+        const amountText = self.add.text(positionX - 15, positionY, '' + newAmount + '/' + maxAmount, { fontSize: '24px' }).setTint(0xffffff).setDepth(100);
+        myGroup.add(missileSprite);
+        myGroup.add(amountText);
+        self.missileCounterUIs[id] = myGroup;
+
+        //wait until either display time is up or the next one appears.
+        let timer = 0;
+        let cleanLoop = null;
+        cleanLoop = setInterval(() => {
```

Comment:
Do we need to clear the ammo count display at all? I feel like we could just have it permanently over the player sprite.

Reply:
sure

Reply:
Sure, I'll bring it up on slack.